README.md

```diff
@@ -21,6 +21,12 @@ npm install
 
 Use `npm run watch` to compile and serve the dist directory in real time.
 
+
+## Add new logo images for sponsors and partners
+
+- Edit image remove background, add whitespace
```

Comment:
We might wanna check the brand guidelines for a given sponsor. Some of them may not like we messing with their logo.
For example: this is the guide for slack: https://slack.com/brand-guidelines

Suggestion: Check for sponsor brand use guidelines before editing the image